x-pack/plugins/security_solution/public/common/components/exceptions/helpers.tsx

```diff
@@ -161,6 +167,10 @@ export const getOperatingSystems = (tags: string[]): string => {
   return osMatches;
 };
 
+export const getOsTagValues = (tags: string[]): string[] => {
```

Comment:
Is this functionally the same as `getOperatingSystems`?

Reply:
I think so. @dplumlee is going to look into dropping the additional formatting from `getOperatingSystems` in the next PR.